spinnaker-jenkins-to-vmss/azuredeploy.json

```diff
@@ -0,0 +1,542 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "spinnakerDnsPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique DNS Name for the Public IP used to access the Spinnaker Virtual Machine."
+            }
+        },
+        "jenkinsDnsPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique DNS Name for the Public IP used to access the Jenkins Virtual Machine."
+            }
+        },
+        "vmUserName": {
+            "type": "string",
+            "metadata": {
+                "description": "Login user name for the spinnaker VM."
+            }
+        },
+        "vmPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Login password for the spinnaker VM."
+            }
+        },
+        "servicePrincipalClientId": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Client ID of the Azure AD application that has rights to spinnaker."
+            }
+        },
+        "servicePrincipalClientSecret": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Client Secret of the Azure AD application that has rights to spinnaker."
+            }
+        },
+        "keyVaultAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of your KeyVault account."
+            }
+        },
+        "OracleUserName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of your Oracle Account used to download the JDK."
+            }
+        },
+        "OraclePassword": {
```

Comment:
Correct, it is passed when we run the initialization script on Jenkins. Removed.